src/meshlab/mainwindow.h

```diff
@@ -507,6 +508,7 @@ private slots:
     QAction *viewFromMeshAct;
     QAction *viewFromRasterAct;
     QAction *viewFromFileAct;
+    QAction *viewToFileAct;
```

Comment:
I would rename this two actions to match the method names

saveViewToFileAct
 readViewFromFileAct

Reply:
I fixed this too.